sys/include/net/gnrc/ndp.h

```diff
@@ -41,14 +41,22 @@
 extern "C" {
 #endif
 
-#define GNRC_NDP_MSG_RTR_TIMEOUT        (0x0210)    /**< Message type for router timeouts */
-#define GNRC_NDP_MSG_ADDR_TIMEOUT       (0x0211)    /**< Message type for address timeouts */
-#define GNRC_NDP_MSG_NBR_SOL_RETRANS    (0x0212)    /**< Message type for multicast
-                                                     *   neighbor solicitation retransmissions */
-#define GNRC_NDP_MSG_RTR_ADV_RETRANS    (0x0213)    /**< Message type for periodic router advertisements */
-#define GNRC_NDP_MSG_RTR_ADV_DELAY      (0x0214)    /**< Message type for delayed router advertisements */
-#define GNRC_NDP_MSG_RTR_SOL_RETRANS    (0x0215)    /**< Message type for periodic router solicitations */
-#define GNRC_NDP_MSG_NC_STATE_TIMEOUT   (0x0216)    /**< Message type for neighbor cache state timeouts */
+/** Message type for router timeouts */
+#define GNRC_NDP_MSG_RTR_TIMEOUT                (0x0210)
+/** Message type for address timeouts */
+#define GNRC_NDP_MSG_ADDR_TIMEOUT               (0x0211)
+/** Message type for multicast neighbor solicitation retransmissions */
+#define GNRC_NDP_MSG_NBR_SOL_RETRANS            (0x0212)
+/** Message type for periodic router advertisements */
+#define GNRC_NDP_MSG_RTR_ADV_RETRANS            (0x0213)
+/** Message type for delayed router advertisements */
+#define GNRC_NDP_MSG_RTR_ADV_DELAY              (0x0214)
+/** Message type for delayed router advertisements in a 6LoWPAN*/
+#define GNRC_NDP_MSG_RTR_ADV_SIXLOWPAN_DELAY    (0x0215)
```

Comment:
When or why?

When: after a short random delay after receiving a RTR SOL.

Why: Because the existing types would cause a periodic retransmission.